modules.d/90multipath/module-setup.sh

```diff
@@ -122,7 +122,7 @@ install() {
         fi
     }
 
-    [[ $hostonly ]] || {
+    [[ $hostonly ]] || mpathconf_installed || {
```

Comment:
I guess technically the conditional below for installing `multipathd-configure.service` is also gated on the `systemd` module being installed. Should we be extra strict here and also check for it in this conditional?

Reply:
That would be an extra PR. It surely makes no sense to install `.service` files without systemd, but it doesn't do much harm either. As @bmarzins already said, mpathconf is used only in Red Hat distributions, and they all use systemd as well.

Reply:
Yup, WFM.